shopinvader_product_sale_packaging/views/product_packaging.xml

```diff
@@ -4,8 +4,9 @@
         <field name="model">product.packaging</field>
         <field
             name="inherit_id"
-            ref="sale_by_packaging.product_packaging_tree_view_inherit"
+            ref="sell_only_by_packaging.product_packaging_tree_view_inherit"
         />
+        <!-- TODO: group will be moved (to shopinvader_restapi) after split -->
```

Comment:
Following work done [here](https://github.com/shopinvader/odoo-shopinvader/pull/1382/), Shopinvader Manager group will be moved to `shopinvader_restapi`. But we normally don't need to depend on `shopinvader_restapi`